src/components/MewSheet/MewSheet.vue

```diff
@@ -1,31 +1,31 @@
-    <!--
-  =====================================================================================
-    Mew Sheet 
-    TODO: need to come back to this to add dark mode and figure out if we actually 
-    need this sheet
-    slot: content (used to place custom ui in the body)
-  =====================================================================================
-  -->
+<!--
+=====================================================================================
+  Mew Sheet 
+  TODO: need to come back to this to add dark mode and figure out if we actually 
```

Comment:
can you remove this TODO comment since you added the dark mode box shadow? thank you! 
also remove the "content" named slot since that was removed as well

Reply:
Fixed